HotelReservation/HotelReservationTests/ReservationSteps/StepsExecutorTests.cs

```diff
@@ -10,7 +10,7 @@
 namespace HotelReservationTests.ReservationSteps
 {
     [TestFixture]
```

Comment:
Właśnie zauważyłem, że masz tutaj testy dla każdego typu stepów. Nie jest to konieczne. Ta klasa pracuje z IReservationStep właśnie po to, żeby nie przejmować się konkretnymi realizacjami tego interfejsu

Reply:
Fixed